Documentation/project-docs/pull-request-validation.md

```diff
@@ -0,0 +1,16 @@
+In the .NET Platform repository we agreed on the following policies before a Pull Request can be merged:
```

Comment:
This repo name doesn't exist yet.  Let's keep it as corefx for now.  We'll need to update a lot of docs in the new merged repo, much more than just this.